src/osd/ECBackend.cc

```diff
@@ -2126,10 +2124,6 @@ bool ECBackend::try_reads_to_commit()
       messages.push_back(std::make_pair(i->osd, r));
     }
   }
-  jspan span;
-  if (op->client_op) {
-    span = tracing::osd::tracer.add_span("EC sub write", op->client_op->osd_parent_span);
-  }
```

Comment:
this removes a bunch of osd spans, but doesn't replace them with anything. didn't we discuss using log events instead of spans here?

Reply:
there were no logs in the original spans, so there is nothing to replace with.
the sole purpose of these spans was to measure the time it takes to execute these functions.

Reply:
right, so don't we need log entries to make those timestamps available?

Reply:
i don't think that ceph users really care about individual function calls timestamps.

Reply:
tracing is for developers too! if we're making choices that remove functionality, please make sure the osd team has a say

Reply:
i brought this up in today's performance call; @jdurgin said it was okay to remove these for now 👍